web-services-app/src/main/java/edu/unc/lib/boxc/web/services/processing/DownloadImageService.java

```diff
@@ -14,11 +14,17 @@
 import java.util.Objects;
 
 /**
+ * Service to process access copy image downloads
  * @author snluong
  */
 public class DownloadImageService {
     private String iiifBasePath;
+    public static final String FULL_SIZE = "full";
+    public static final String INVALID_SIZE_MESSAGE = "Unable to determine size for access copy download";
 
+    /*
```

Comment:
javadoc begins with `/**`, if you do that and hit enter, intellij will fill in all the boilerplate stuff for you